README.md

```diff
@@ -16,7 +16,11 @@ depend on the number of vertices, the number of edges, or both?
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
 
+The runtime complxeity should depend on both edges and vertices as it should scale with V to create a V by V matrix, so $\Theta(V^2)$. The actual converting from list to matrix and filling the 0s with 1s, it should be $\Theta(V*E)$ time as for each vertices, it only checks the edges and then replaces the numbers. It does not actually check every possible position in the matrix so it can not be $V^2$.
```

Comment:
Does it check all edges for each vertex?

Reply:
it doesnt check every edge for each vertex, rather it goes to each vertex and for each vertex it checks the same vertex in the adj list to see which columns need to be swapped and swaps them. Would this actually be V+E instead of V*E as its going through V vertex a total of E edge times?

Reply:
That sounds like it's going in the right direction!